_includes/releases/v21.2/v21.2.15.md

```diff
@@ -0,0 +1,49 @@
+## v21.2.15
+
+Release Date: August 29, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v21-2-15-sql-language-changes">SQL language changes</h3>
+
+- Removed the `DatabaseID` field from the sampled query telemetry log due to its potential to cause indefinite blocking in the case of a lease acquisition failure. [#85022][#85022]
+
+<h3 id="v21-2-15-operational-changes">Operational changes</h3>
+
+- Telemetry logs now display more finely redacted error messages from SQL execution. Previously, the entire error string was redacted. [#85463][#85463]
```

Comment:
For lines 7 - 13: I have been instructed in the past to remove sampled query / telemetry change log entries from these RNs. I think someone would want this removed here also, but I don't know for sure.